rewrite-test/build.gradle.kts

```diff
@@ -17,5 +17,5 @@ dependencies {
 
     testImplementation(project(":rewrite-groovy"))
     testRuntimeOnly("org.antlr:antlr4-runtime:4.13.2")
-    testRuntimeOnly(project(":rewrite-java-21"))
+    testRuntimeOnly(project(":rewrite-java-25"))
```

Comment:
Some downstream projects rely on this transitive dependency for their Java parser. I wonder what using 25 here will do for their tests when those concern using gradle, but I'll guess we'll see over the next hours and otherwise change this back.